### ISSUE TITLE
Status of Notes

### ISSUE THREAD

**Issue by Lady-Palmac** [anthonysanlucas/pinned-notes#1]
I think that states should be added to notify whether the assignment of each argument is complete or pending.

**Comment by anthonysanlucas** [comment]
Good idea, however, this tool is not intended to become an Everything...

What would be the states?  I think it could be useful:

- Completed
- On Hold
- In Process
- Review
- No status

**Comment by Lady-Palmac** [comment]
In this case it would be:
- Completed 
- In process
- Review

**Comment by anthonysanlucas** [comment]
Those states make sense, I'll put it on my list so I can do it, thank you very much for the idea.
